testbed/tests/test_window.py

```diff
@@ -226,6 +226,28 @@ async def test_secondary_window_with_args(app, second_window, second_window_prob
 
         assert second_window not in app.windows
 
+    async def test_instantiate_window_with_content(app):
+        """A window can be created with content"""
+        try:
+            window_no_content = toga.Window()
+            assert window_no_content.content is None
+        finally:
+            window_no_content.close()
+            window_no_content = None
+
+        # Setup the box with something inside it:
+        label1 = toga.Label("Hello World")
+        content = toga.Box(children=[label1])
+        try:
+            window_with_content = toga.Window(content=content)
+            assert window_with_content.content == content
+            # Make sure it still has that internal content:
+            assert len(window_with_content.content.children) == 1
+            assert window_with_content.content.children[0] == label1
```

Comment:
To verify this is working, the window needs to be shown, and a probe needs to wait for the window to be displayed so that a `--slow` invocation can visually verify the result.